src/sync/sync_permission.cpp

```diff
@@ -170,16 +171,22 @@ void Permissions::set_permission(std::shared_ptr<SyncUser> user,
                                  PermissionChangeCallback callback,
                                  const ConfigMaker& make_config)
 {
+    using namespace std::chrono;
     const auto realm_url = user->server_url() + permission.path;
     auto realm = Permissions::management_realm(std::move(user), make_config);
     CppContext context;
 
+    // Get the current time.
+    int64_t ns_since_epoch = duration_cast<nanoseconds>(system_clock::now().time_since_epoch()).count();
+    int64_t s_arg = ns_since_epoch / (int64_t)Timestamp::nanoseconds_per_second;
+    int32_t ns_arg = ns_since_epoch % Timestamp::nanoseconds_per_second;
+
     // Write the permission object.
     realm->begin_transaction();
     auto raw = Object::create<util::Any>(&context, realm, *realm->schema().find("PermissionChange"), AnyDict{
         { "id", util::uuid_string() },
-        { "createdAt", Timestamp(0, 0) },
-        { "updatedAt", Timestamp(0, 0) },
+        { "createdAt", Timestamp(s_arg, ns_arg) },
+        { "updatedAt", Timestamp(s_arg, ns_arg) },
```

Comment:
Is it possible for `createdAt` and `updatedAt` to have different values? How would that come about? If it's not, why do we have both of them?